torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3676,6 +3676,27 @@ def generator():
     return list(generator())
 
 
+def sample_inputs_resize_ops(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, dtype=dtype, device=device)
+    cases = (((S, S, S), (S * S, S)),
+             ((), ()),
+             ((), (1, 1, 1)),
+             )
```

Comment:
Style nit (do not fix this): this paren is mismatched

Reply:
With
```python
cases = (((S, S, S), (S * S, S)),
         ((), ()),
         ((), (1, 1, 1)),
        )
```
we get
```
flake8 torch/testing/_internal/common_methods_invocations.py 
torch/testing/_internal/common_methods_invocations.py:3684:13: E124 closing bracket does not match visual indentation
```

Reply:
Huh; weird. Oh well